hack/pre-release.sh

```diff
@@ -63,11 +64,13 @@ update_WMCO_version() {
   echo "Updating the skipRange version in $cluster_service_version_yaml to $updated_version"
   # Example: matches olm.skipRange: '>=8.0.0 *<9.0.0*'
   sed -i "s/<$version/<$updated_version/g" $cluster_service_version_yaml
-
-  echo "Updating previous skipRange version in $cluster_service_version_yaml to $version"
-  # Example: matches olm.skipRange: '*>=8.0.0* <9.0.0'
-  sed -i "s/>=$last_version/>=$version/g" $cluster_service_version_yaml
- 
+  
+  if [[ $xyz_stream == "x" ]]; then 
+    echo "Updating previous skipRange version in $cluster_service_version_yaml to $version"
```

Comment:
This message is confusing. What do you mean by `previous skipRange version`? I realize this issue was there before but this is bringing it more to light. You need to clearly call out what is being updated here vs the previous `Updating the skipRange version in` message i.e. lower range vs upper range.